guides/plugins/apps/storefront/apps-as-themes.md

```diff
@@ -24,4 +24,4 @@ If your app provides a `theme.json` file, it is considered to be a theme. All th
 
 ## Migrating existing themes
 
-If you already created Shopware 6 themes via plugins, it is effortless to migrate them to the app system. Don't worry - you don't have to do all work twice. Instead of providing a `composer.json` and plugin base class, provide a `manifest.xml` file with the metadata for your app. After you created a new folder for your app and added the `manifest.xml`, you can copy the `YourThemePlugin/src/Resources` folder from your plugin to the `YourThemeApp/Resources` folder inside your app. It should not be necessary to change anything inside your template or Javascript code at all.
+If you already created Shopware 6 themes via plugins, it is effortless to migrate them to the app system. Don't worry—you don't have to do all work twice. Instead of providing a `composer.json` and plugin base class, provide a `manifest.xml` file with the metadata for your app. After you created a new folder for your app and added the `manifest.xml`, you can copy the `YourThemePlugin/src/Resources` folder from your plugin to the `YourThemeApp/Resources` folder inside your app. It should not be necessary to change anything inside your template or Javascript code at all.
```

Comment:
```suggestion
If you have already created Shopware 6 themes via plugins, it is effortless to migrate them to the app system. Don't worry, you don't have to do all the work twice. Instead of providing a `composer.json` and plugin base class, provide a `manifest.xml` file with the metadata for your app. After you have created a new folder for your app and added the `manifest.xml`, you can copy the `YourThemePlugin/src/Resources` folder from your plugin to the `YourThemeApp/Resources` folder inside your app. It should not be necessary to change anything inside your template or Javascript code.
```